react/week3/todo3/src/Components/TodoForm.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from "react";
+
+class TodoForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { todoInput: "" };
+  }
```

Comment:
You have made this component as a Class component. 
Was this on purpose? 
I'd go for all functional components if I were you :)